src/java.base/share/classes/sun/util/locale/LocaleMatcher.java

```diff
@@ -148,33 +156,6 @@
         return list;
     }
 
-    /**
-     * Removes the tag(s) which are falling in the basic exclusion range(s) i.e
-     * range(s) with q=0 and returns the updated collection. If the basic
-     * language ranges contains '*' as one of its non zero range then instead of
-     * returning all the tags, remove those which are matching the range with
-     * quality weight q=0.
-     */
```

Comment:
Please keep the comments in the modified code (except the `*` part). Although it's OK to remove this method as it eliminates extra `ArrayList` allocation, please keep the comments in the modified code (except the `*` part).

Reply:
I've tried to follow your suggestion but I think this comment is no longer necessary. (The tags are no longer removed and no collections are updated.)

I had noticed that this comment could not be added back verbatim to the modified code. Hence, I tried to rewrite it. However, I noticed that the comments for `LocaleMatcher{shouldIgnoreFilterBasicMatch,shouldIgnoreFilterExtendedMatch}` clearly covers what we wanted to highlight here.

Let me know if you disagree together with a comment that you'd like me to include verbatim.